src/pages/researcher.pug

```diff
@@ -0,0 +1,90 @@
+extends ../layouts/case-study.pug
+
+block prepend vars
+  -var pageTitle = "Researcher job - Leaf"
+  -var metaUrl = "https://weareleaf.com/researcher"
+  -var metaTitle = "Researcher"
+  -var metaDescription = "We're looking for a researcher to help drive change in our 4-year old our company, from how we target and win new business, to how we present ourselves to the World."
+
+block pageComponents
+  +banner({
+    bannerCompany: "leaf",
+    bannerHeading: "Researcher"
+  })
+
+  .section.section--sm-spacing
+    .container
+      +intro({
+        introImage: {
+          src: "/assets/images/researcher/logo.svg",
+          alt: "Leaf",
+        },
+        introText: "We're looking for a researcher to drive exciting changes at Leaf, helping us deliver on a more mature lead-generation strategy, and enhancing the value we can offer our clients."
```

Comment:
Is it better to use the metaDescription value here for consistency/SEO purposes?